manifests/reposync-crd.yaml

```diff
@@ -1296,6 +1296,26 @@ spec:
                     format: int64
                     minimum: 0
                     type: integer
+                  logLevel:
+                    description: logLevel allows one to override the log verbosity
+                      of a container. ContainerLogLevelOverride is a struct with field
```

Comment:
ContainerLogLevelOverride is an internal name. It doesn't need to be in the public description here. How about `Each entry must contain the name of a reconciler deployment container and the desired log level.`

Reply:
changed